versions/i-/irrlicht.json

```
@@ -1,7 +1,12 @@
 {
   "versions": [
     {
-      "git-tree": "e2b2dd836e693401dab067c5064665e926a46b07",
+      "git-tree": "18ae08d6099b5457ccedbdb7209c79321a9e9027",
+      "version-string": "1.8.4",
+      "port-version": 11
+    },
+    {
+      "git-tree": "da996239bd0a673f57f59686d2fdf350e9b5ea64",
```

This should not be changed. You can run `.\vcpkg x-add-version --overwrite-version simage` to update this.
```suggestion
      "git-tree": "e2b2dd836e693401dab067c5064665e926a46b07",
```

---

i am not entirely sure why this should *not* change - i do not update the versions manually, just running `vcpkg x-add-version --overwrite-version irrlicht` **after the commit**.